pytorch_lightning/strategies/strategy.py

```diff
@@ -52,6 +52,7 @@ def __init__(
         self.checkpoint_io = checkpoint_io
         self.precision_plugin = precision_plugin
         self.optimizers: List[Optimizer] = []
+        # FIXME: rename to _config?
```

Comment:
There's no need to rename this in this PR, but do we want to?

Reply:
+1 for renaming it to `lr_scheduler_configs`. Should we have consistent variable names in the codebase because currently, variable name `lr_scheduler(s)` is used for both configs and actual lr schedulers.

Reply:
Yes. Would you like to help with that after this PR is merged?

Reply:
Yes!

Reply:
@akihironitta You are good to start this now. No need to deprecate because strategies are currently experimental so should be a simple rename